api/tests/unit/domain/services/recommendation-service_test.js

```diff
@@ -0,0 +1,112 @@
+const { expect } = require('../../../test-helper');
+
+const recommendationService = require('./../../../../lib/domain/services/recommendation-service');
+
+describe('Unit | Service | Campaign Recommendation Service', () => {
+  describe('computeRecommendationScore', () => {
+
+    context('when no skills validated', () => {
+      it('should return 30', () => {
```

Comment:
le it est pas révélateur de ce qu'on test. Peut être ajouter un test qui met en évidence de pas avoir le score max de la difficulté.

Reply:
Done